taqueria-vscode-extension/src/lib/gui/helpers/TzKTFetcher.ts

```diff
@@ -0,0 +1,97 @@
+import fetch from 'node-fetch-commonjs';
+
+export type TzKtHeadData = {
+	chain: string;
+	chainId: string;
+	cycle: number;
+	level: number;
+	hash: string;
+	protocol: string;
+	nextProtocol: string;
+	timestamp: string;
+	votingEpoch: number;
+	votingPeriod: number;
+	knownLevel: number;
+	lastSync: string;
+	synced: true;
+	quoteLevel: number;
+	quoteBtc: number;
+	quoteEur: number;
+	quoteUsd: number;
+	quoteCny: number;
+	quoteJpy: number;
+	quoteKrw: number;
+	quoteEth: number;
+	quoteGbp: number;
+};
+
+export type TzKTAccountData = {
+	type: 'delegate' | 'contract';
+	address?: string | null;
+	alias?: string | null;
+	balance?: number | null; // int64
+};
+
+export type TzKTContractEntrypointData = {
+	name: string;
+	jsonParameters?: any;
```

Comment:
Is the question mark necessary for `any` type? I thought `any` covers `undefined` too, right?

Reply:
Question marks makes property optional, so you don't need to provide it by `foo: undefined` or so